components/org.wso2.carbon.identity.sso.saml/src/main/java/org/wso2/carbon/identity/sso/saml/servlet/SAMLSSOProviderServlet.java

```diff
@@ -792,7 +808,6 @@ private void sendToFrameworkForAuthentication(HttpServletRequest req, HttpServle
         String sessionDataKey = UUIDGenerator.generateUUID();
         addSessionDataToCache(sessionDataKey, sessionDTO);
 
-        String commonAuthURL = IdentityUtil.getServerURL(FrameworkConstants.COMMONAUTH, false, true);
         String selfPath = req.getContextPath();
```

Comment:
Shouldn't this path be tenant qualified in the tenant qualified URL mode?

Reply:
It's not used

Reply:
I was referring to the 'selfPath' attribute that goes in request to the commonauth

Reply:
Noted. Actually, for SAML it was not utilized. There was no redirect back from frame work for saml endpoint. 
This happened as credentials were posted to /samlsso endpoint which internally delegated those to the framework, that avoided a redirection to handle protocol specific response upon authentication.

Reply:
Fixed this path as well, considering the fact that there can be a case where someone would have posted credentials to the /commonauth endpoint. 
In such cases, caller path being set here will be used to redirect back to /samlsso to build the SAML response to the application.